### PR TITLE
Allow attaching same external load balancer to multiple instance groups

### PR DESCRIPTION
**Pull request by hakman**
Similar to https://github.com/kubernetes/kops/pull/10335
Fixes: https://github.com/kubernetes/kops/issues/10664

/cc @rifelpet @rdrgmnzs

### CODE REVIEW

File: tests/integration/update_cluster/externallb/kubernetes.tf [hunk 1]
```diff
@@ -86,7 +86,7 @@ resource "aws_autoscaling_group" "master-us-test-1a-masters-externallb-example-c
     id      = aws_launch_template.master-us-test-1a-masters-externallb-example-com.id
     version = aws_launch_template.master-us-test-1a-masters-externallb-example-com.latest_version
   }
-  load_balancers      = ["my-external-elb-2", "my-external-elb-3"]
+  load_balancers      = ["master-us-test-1a.masters.externallb.example.com-my-external-elb-1", "master-us-test-1a.masters.externallb.example.com-my-external-elb-2", "master-us-test-1a.masters.externallb.example.com-my-external-elb-3"]
```
Comment (rifelpet) [inline tests/integration/update_cluster/externallb/kubernetes.tf:89]: I wouldn't expect the terraform and cloudformation outputs to be changing. We don't want to change the names of the load balancers we're attaching to the ASGs, only the names of their fi tasks

Comment (hakman) [reply tests/integration/update_cluster/externallb/kubernetes.tf]: True. These latest changes should do the trick.
I guess now this has to its way into 1.19 and 1.18?

**Pull request merged**